### ISSUE TITLE
Error on running build "TypeError: Cannot read properties of undefined"

### ISSUE THREAD

**Issue by awinabi** [serverless-stack/open-next#13]
I run `npx open-next@latest build` on a nextjs project and I get the error `Bundling image optimization function...` step

```
Bundling server function...
Bundling image optimization function...
file:///Users/awin/.npm/_npx/a94ce00fc9d9ef90/node_modules/open-next/dist/build.js:31
    createMiddlewareBundle(ret.output.middleware.files["index.js"].data);
                                                 ^

TypeError: Cannot read properties of undefined (reading 'files')
    at build (file:///Users/awin/.npm/_npx/a94ce00fc9d9ef90/node_modules/open-next/dist/build.js:31:50)

```

full error log at https://gist.github.com/awinabi/fb718f57bda3a9bbb451f02811931b42

I'm running the latest version of open-next.

Do I have to do some explicit config for the middleware functions?

**Comment by khuezy** [comment]
Looks like `nextBuild` output for middleware is optional. OpenNext needs to check for this before creating the middleware bundle.
Can you try adding some mock middleware.ts file in your nextapp to see if the error goes away?

**Comment by fwang** [comment]
Thanks @khuezy.

Hey @awinabi this is fixed in the latest v0.4.0 release.

**Comment by awinabi** [comment]
Thank you for the quick response. The latest version of open-next solves it.
